src/locale/locales/pt-BR/messages.po

```diff
@@ -7394,7 +7394,7 @@ msgstr "Dessilenciar posts com {displayTag}"
 
 #: src/components/dms/ConvoMenu.tsx:176
 msgid "Unmute conversation"
-msgstr ""
+msgstr "Dessilencia conversa"
```

Comment:
In this context, it would be more appropriate to use "desmutar" instead of "dessilencia". The term "desmutar" is a recognized word in Portuguese and clearly conveys the idea of reversing a state of silence or resuming communication. "Dessilencia", on the other hand, is not a common term and could cause confusion. Therefore, for greater clarity and accuracy, "desmutar conversa" is a better choice for translating "Unmute conversation".

Reply:
OK - Desmutar conversa